test/e2e/apimachinery/watch.go

```diff
@@ -41,7 +41,13 @@ const (
 var _ = SIGDescribe("Watchers", func() {
 	f := framework.NewDefaultFramework("watch")
 
-	It("should observe add, update, and delete events on pods", func() {
+	/*
+		    Testname: watch-pods-with-label-selector
+		    Description: Ensure that multiple watchers are able to receive all add,
+			update, and delete events on pods that match a label selector and do
+			not receive events for pods which do not match that label selector.
+	*/
+	framework.ConformanceIt("should observe add, update, and delete events on pods", func() {
```

Comment:
I think for the purpose of conformance tests we need a bit more than what is here.

The scenarios that I would additionally see tested are:

1. check if starting the same watch once all changes are already there will result in the same event (for that purpose we need to start the watch from a given resource version)

2. check if then restarting the watch on RV on which the previous finished will continue delivering events from that point in time.

Reply:
Thank you, this is a very helpful suggestion.

Reply:
So the test you added addresses the first point, but not necessary the second one.

I'm fine with the PR itself, but I'm not convinced we should already mark these tests as conformance.
In other words, I'm fine with merging this PR if you:
- change ConformanceIt to IT
- add a TODO describing the second scenario we want to test